content/en/platform/corda/5.1/deploying-operating/observability/metrics/task-manager.md

```diff
@@ -0,0 +1,57 @@
+---
+date: '2023-10-26'
+version: 'Corda 5.1'
+title: "Task Manager"
+menu:
+  corda51:
+    parent: corda51-cluster-metrics
+    identifier: corda51-cluster-task-manager
+    weight: 1450
+section_menu: corda51
+---
+
+# Task Manager
+
+The task manager exposes metrics for underlying thread pools and jobs processing duration and count.
+
+<style>
+table th:first-of-type {
+    width: 25%;
```

Comment:
I'd make this width bigger as it's the only column that can't break gracefully. You could then make tags and description narrower.

Reply:
Leaving as is to match other pages in this "metrics" section.